service/iot-service-client/pom.xml

```diff
@@ -80,6 +80,16 @@
             <artifactId>slf4j-api</artifactId>
             <version>1.7.25</version>
         </dependency>
+        <dependency>
+            <groupId>com.azure</groupId>
+            <artifactId>azure-core</artifactId>
+            <version>1.12.0</version>
+        </dependency>
+        <dependency>
+            <groupId>com.azure</groupId>
+            <artifactId>azure-core-amqp</artifactId>
```

Comment:
is this used only for the enum? if the actual amqp impl is still from proton-j, then adding that as a comment here would be useful

Reply:
This dependency is now TBD, we may just remove this dependency and create this enum in our repo, as per earlier discussion

Reply:
This enum has since been removed since we no longer need to distinguish between types of tokens from the TokenCredential instance